text/src/autogluon/text/automm/data/process_text.py

```diff
@@ -35,6 +41,59 @@
 }
 
 
+def construct_augmenter(
+        augment_types: List[str],
+) -> Optional[naf.Sometimes]:
+    """
+    Build up a text augmentor from the provided list of augmentation types
+    
+    Parameters
+    ----------
+    augment_types
+        A list of text augment types.
+
+    Returns
+    -------
+    A nlpaug sequantial flow.
+    
+    """
+    if(augment_types is None or len(augment_types) == 0):
+        return None
+
+    auglist = []
+    try:
+        nltk.data.find('tagger/averaged_perceptron_tagger')
+    except LookupError:
+        nltk.download('averaged_perceptron_tagger')
+    for aug_type in augment_types:
+        if '(' in aug_type:
+            trans_mode = aug_type[0:aug_type.find('(')]
+            kwargs = ast.literal_eval(aug_type[aug_type.find('('):])
+        else:
+            trans_mode = aug_type
+            kwargs = {}
+        if trans_mode == "synonym_replacement": 
+            kwargs['aug_src'] = 'wordnet'
```

Comment:
I recommend to use `"` instead of `'` to be consistent with the existing code.

Reply:
It should be the same. We may change it after we have adopted black.

Reply:
Although they are both correct logically, the existence of both does affect the code quality, which is important to attract potential contributors and grow community.

Reply:
Let’s go to this type of details after we adopted pylint for lint check, or have adopted black. It’s okay for now.

Reply:
Basically, paying too much attention to coding style without a specific lint tool/style guidance is not efficient. You can read pep8 to see a reference style guidance: https://peps.python.org/pep-0008/ . From my point of view, we should focus on enabling lint check as soon as possible instead of ensuring the coding style via code review, which is super inefficient.

Reply:
After we enable pylink, can pylink automatically correct the code already merged? If not, we'd better fix `"` before merging.

Reply:
I think pylint is able to check both existing and new code, depending on how we use it. I agree with @sxjscience  that we should prioritize lint check before other features. This is very important for such a big project with so many modules and new PRs almost every day.